Makefile

```diff
@@ -13,7 +13,7 @@ all:
 # deps will clean out the vendor directory and use go mod for a fresh install
 deps:
 	rm -rf vendor
-	go mod vendor
+	go mod vendor -v
 	go mod tidy
```

Comment:
Why do we need to print them to standard error? If the reason is to log possible package download issues, does it make sense to also add the `-v` flag to `go mod tidy` ? Again, just wondering

Reply:
You are right, for consistency, better to add the `-v` on `tidy` if we keep it on `vendor`. i kind of like it so we know if anything goes bad. Let me know if you prefer not to have the verbose mode.